tutorial-running-chaos-workflow-on-remote-cluster/your-first-pwapp/index.html

```diff
@@ -0,0 +1,369 @@
+
+<!doctype html>
+
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="theme-color" content="#4F7DC9">
+  <meta charset="UTF-8">
+  <title>Running Chaos Workflows on Remote Clusters</title>
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400|Roboto:400,300,400italic,500,700|Roboto+Mono">
+  <link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://storage.googleapis.com/codelab-elements/codelab-elements.css">
+  <style>
+    .success {
+      color: #1e8e3e;
+    }
+    .error {
+      color: red;
+    }
+  </style>
+</head>
+<body>
+  <google-codelab-analytics gaid="UA-49880327-14"></google-codelab-analytics>
+  <google-codelab codelab-gaid=""
+                  id="your-first-pwapp"
+                  title="Running Chaos Workflows on Remote Clusters"
+                  environment="web"
+                  feedback-link="">
+    
+      <google-codelab-step label="Introduction" duration="1">
+        <p>Chaos workflows cannot only be scheduled on your default cluster, Litmus allows you to schedule chaos workflow on a remote cluster.</p>
+<p>In this tutorial, you will learn how to create chaos workflows on a remote cluster. </p>
+<p>First, you will learn to deploy your own remote cluster on GKE(Google Kubernetes Engine), then you will install a CLI tool called <a href="https://github.com/litmuschaos/litmusctl" target="_blank">litmusctl</a>. Next, you will use <a href="https://github.com/litmuschaos/litmusctl" target="_blank">litmusctl</a> to target this remote cluster agent. Finally, you will schedule a predefined workflow on this remote cluster.</p>
+<h2 is-upgraded>What you will learn</h2>
+<ul>
+<li>How to deploy a remote cluster to GKE</li>
+<li>Litmusctl</li>
+<li>How to connect to an external Agent</li>
+<li>How to schedule a workflow on a remote cluster</li>
+</ul>
+<h2 is-upgraded>Prerequisites</h2>
+<ul>
+<li>Minikube 1.2.0 or later. Recommended sizing of at least: 2 vCPUs, 8GB memory, 10GB of disk space (for persistent volume creation)</li>
```

Comment:
Can we change this to be with respect to version of kubectl.
Like here https://litmuschaos.github.io/tutorials/tutorial-getting-started/index.html#0
Users can try this tutorial on other clusters also, so we should not mention Minikube.